daemon/container_unix.go

```diff
@@ -981,9 +985,11 @@ func (daemon *Daemon) connectToNetwork(container *Container, idOrName string, up
 		return err
 	}
 
-	createOptions, err := container.buildCreateEndpointOptions(n)
-	if err != nil {
-		return err
+	if createOption {
```

Comment:
Can you check whether this function can derive it is being called for the first network connect ?
Something like:

``
if len(container.NetworkSettings.Networks) == 1 {
     program the create option
}
``

If so, caller does not need to pass any bool